packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -101,6 +101,14 @@ function ExploreTemplates() {
       .filter(composeTemplateFilter(selectFilters));
   }, [templatesOrderById, templates, selectFilters]);
 
+  const totalVisibleTemplates = useMemo(
```

Comment:
This fixes the no results empty view of templates now that it searches. 

Since it's a static search  - no actual updates to the data in state - I opted to just memoize a check here with the totalTemplates vs the orderedTemplates (which get filtered). 

![Screen Shot 2021-11-18 at 2 45 27 PM](https://user-images.githubusercontent.com/10720454/142660578-2319a95b-eaae-4421-8e21-7e688baacdd9.png)

Reply:
ope. good find!